crates/evm/fuzz/src/strategies/invariants.rs

```diff
@@ -65,18 +67,20 @@ pub fn invariant_strat(
 ) -> impl Strategy<Value = BasicTxDetails> {
     let senders = Rc::new(senders);
     any::<prop::sample::Selector>()
-        .prop_flat_map(move |selector| {
-            let contracts = contracts.targets.lock();
-            let functions = contracts.fuzzed_functions();
-            let (target_address, target_function) = selector.select(functions);
-            let sender = select_random_sender(&fuzz_state, senders.clone(), dictionary_weight);
-            let call_details = fuzz_contract_with_calldata(
-                &fuzz_state,
-                &fuzz_fixtures,
-                *target_address,
-                target_function.clone(),
-            );
-            (sender, call_details)
+        .prop_flat_map({
+            move |selector| {
+                let contracts = contracts.targets.lock();
+                let functions = contracts.fuzzed_functions();
+                let (target_address, target_function) = selector.select(functions);
+                let sender = select_random_sender(fuzz_state.clone(), senders.clone(), dictionary_weight);
+                let call_details = fuzz_contract_with_calldata(
+                    fuzz_state.clone(),
+                    fuzz_fixtures.clone(),
```

Comment:
Due to lifetime / ownership issues `clone` is not required here - not ideal!

Reply:
you mean, is required?
isn't this giga expensive?

Reply:
Sorry - yes, is required